README.md

```diff
@@ -1,139 +1,32 @@
 # Kritis
 
-Kritis (“judge” in Greek), provides full software supply chain security for Kubernetes applications,
-allowing devOps teams to enforce deploy-time image security policies using metadata and attestations stored in [Grafeas](https://github.com/grafeas/grafeas).
+Kritis (“judge” in Greek), is an open-source solution for software supply chain security for Kubernetes applications. Kritis enforces deploy-time security policies using the [Google Cloud Container Analysis API](https://cloud.google.com/container-analysis/api/reference/rest/), and in a subsequent release, [Grafeas](https://github.com/grafeas/grafeas).
 
-You can read the [Kritis whitepaper](https://github.com/Grafeas/Grafeas/blob/master/case-studies/binary-authorization.md) for more details.
+Here is an example Kritis policy, to prevent the deployment of Pod with a critical vulnerability unless it has been whitelisted:
 
```

Comment:
Critical in caps maybe  or inside `Critical` ?

Reply:
I had CRITICAL at first, but decided that it did not help readability, as by this point of the intro, we haven't yet introduced severity levels. The casual adjective form of critical seemed easier to swallow.

Without the introduction, CRITICAL is also specifically confusing since the example YAML says HIGH, rather than CRITICAL. 

On the other hand, capitalizing Critical makes it seem like a proper API object.